section3/exercises/arrays.js

```diff
@@ -107,10 +126,18 @@ var chargerType = "car";
 
 // Write your conditional here
 
+//Establishes if the user meets the minimum requirments
+  if (percentBatteryLeft >= 15){
+    console.log("Congrats, you have enough battery to order an Uber!");
+//Section will run if the user does not have the propper amount of battery
```

Comment:
What about the case in the directions when the user has "a charger and it is a car charger" ? Please update your code to console log the proper message in this case.